db/seeds/development.rb

```diff
@@ -0,0 +1,78 @@
+# This file should contain all the record creation needed to seed the database with its default values.
+# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
+#
+# Examples:
+#
+#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
+#   Mayor.create(name: 'Emanuel', city: cities.first)
+
+PICTURE = "clam_default.png"
+
+10.times do
+  Item.create(title: Faker::Commerce.product_name,
+              description: Faker::Hacker.say_something_smart,
+              image: PICTURE,
+              price: Faker::Number.number(4),
+              item_status_id: 1)
+end
+
+5.times do
+  Item.create(title: Faker::Commerce.product_name,
+              description: Faker::Hacker.say_something_smart,
+              image: PICTURE,
+              price: Faker::Number.number(4),
+              item_status_id: 2)
+end
+
+5.times do
+  Category.create(name:Faker::Commerce.department)
+end
+
+30.times do
+  selected_item = Item.all.sample
+  selected_category = Category.all.sample
+  ItemCategory.create(item_id: selected_item.id,
+                      category_id: selected_category.id)
+end
+
+# Create Users
+User.create(email: 'test@gmail.com', full_name: "test_user", display_name: "test_user", password: "test", role: 0)
```

Comment:
Line is too long. [114/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.